Sources/MapboxMaps/Annotations/Generated/PointAnnotation.swift

```diff
@@ -1,7 +1,7 @@
 // This file is generated.
 import Foundation
 
-public struct PointAnnotation: Annotation {
+open struct PointAnnotation: Annotation {
```

Comment:
@Knapiii Could you elaborate one the motivation behind this change? It's not entirely clear what are you trying to achieve changing the access modifier to `open`.

Reply:
I can not create my own Custom Annotation class and inherit `PointAnnotation`, if `PointAnnotation` is public.
I'm getting the error `Cannot inherit from non-open class 'PointAnnotation' outside of its defining module` if I try.

I want to have my own variables and states on the `PointAnnotation` class

Reply:
I don't think that changing the access modifier to `open` will help to inherit from `PointAnnotation`, in Swift only classes can be subclassed. 

If you want to maintain an extra state for each annotation my suggestion is to have a separate model for it and link it with the corresponding annotation through the `id` property.
e.g.:
```swift
final class MyViewController: UIViewController {
    func viewDidLoad() {
        super.viewDidLoad()
        
        let myModels = [Store(id: "23432-234324", name: "...", location: ..., myCustomProperty: ...)]
        let annotations = myModels.map { model in
            var annotation = PointAnnotation(id: model.id, coordinate: model.location)
            annotation.textField = model.name
            return annotation
        }
    annotationManager.annotations = annotations
    }
}
```

Reply:
@evil159 When I tested and made `PointAnnotation` open, it actually did help me inherit `PointAnnotation`